website/docs/components/tooltip/partials/guidelines/guidelines.md

```diff
@@ -2,34 +2,81 @@
 
 ### When to use
 
-- {description}
+When displaying additional context or information that is not critical to the user's task or decision-making process.
 
 ### When not to use
 
-- {description}, consider {[component](#)}.
-- {description}, consider {[component](#)}.
-- {description}, consider {[component](#)}.
+- When the content is too lengthy or complex.
+- When the information provided is critical to the user's task or decision-making process, consider moving the content to the page or an [alert](https://helios.hashicorp.design/components/alert).
 
-### Variant/property name
+tooltip-when-to-use
 
-<!-- don’t forget to include real examples and do/don’t blocks, as necessary -->
-{description}
+!!! Do
 
-### Variant/property name
+![Correct use of a tooltip](/assets/components/tooltip/tooltip-when-to-use.png)
 
-<!-- don’t forget to include real examples and do/don’t blocks, as necessary -->
-{description}
+!!!
+
+!!! Dont
```

Comment:
[non-blocking] It seems like it's helpful in the do/dont blocks to specifically call out what the recommendation is. Specifically in this scenario it could be interpreted in a couple of different ways: Don't use a tooltip in place of helper/placeholder text, don't use a tooltip on form fields, don't use a tooltip to highlight blocking actions, etc. Probably good for us to align holistically on what goes inside of these blocks other than images/examples.

Reply:
Thanks for the feedback, Jory. This is a good point. I added a short sentence to each block, but I'm unsure if the sentence in the Don't block sounds good or a little bit forced. Wdyt?

`Avoid using tooltips to place crucial information related to the user's task.`

Reply:
Yep, that's feels good to me and is explicit enough